docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -0,0 +1,75 @@
+#%RAML 1.0 DataType
+type: object
+description: How much disk space is needed for this application. This number does not have to be an integer, but can be a fraction.
+properties:
+  id:
+    type: !include pathId.raml
+    description: Unique identifier for the pod consisting of a series of names separated by slashes. Each name must be at least 1 character and may only contain digits (`0-9`), dashes (`-`), dots (`.`), and lowercase letters (`a-z`). The name may not begin or end with a dash.
+    required: true
+    example: /ops/audit
+  containers:
+    type: array
+    items: !include mesosContainer.raml
+  resources:
+    type: !include resources.raml
+    description: The resources to allocate to the pod
+  instances:
+    type: integer
+    description: The number of instances of this pod to start.
+    minimum: 0
+  maxInstances?:
+    type: integer
+    description:|
+      The maximum number of instances of this pod that should
+      be running at a given time.
+    minimum: 0
+  constraints?:
+    type: array
+    items: !include constraint.raml
+    description:|
+      Valid constraint operators are one of UNIQUE, CLUSTER, GROUP_BY, LIKE, UNLIKE, MAX_PER
+    example: [hostname:UNIQUE]
+  user?:
+    type: string
+    description: The user to use to run the tasks on the agent
+  version?:
+    type: datetime-only
+    description: The version of the definition, immutable
+  versionInfo?: !include versionInfo.raml
+  env?:
```

Comment:
should be consistent w/ names. either container.environment and use the environment here, or both should use env